doc/inspect-app-graph.md

```diff
@@ -63,5 +63,37 @@ application.asDotString ====
 
 ![webgraphviz](webgraphviz-example.png)
 
+### With specs2
```

Comment:
Perhaps this could be in its own file, `test.md`, where we can elaborate better regarding testing strategies, in the future?

Reply:
Let's do that if that part expands in the future.